web/src/reactions.js

```diff
@@ -154,18 +154,24 @@ function generate_title(emoji_name, user_ids) {
 
     if (user_ids.length === 1) {
         if (current_user_reacted) {
-            return $t({defaultMessage: "You (click to remove) reacted with {emoji_name}"}, context);
+            return $t_html(
+                {defaultMessage: "You (click to remove) reacted with {emoji_name}"},
+                context,
+            );
         }
         context.username = usernames[0];
-        return $t({defaultMessage: "{username} reacted with {emoji_name}"}, context);
+        return $t_html(
+            {defaultMessage: "<div><span>{username}</span> reacted with {emoji_name}</div>"},
+            context,
+        );
```

Comment:
@SahilSingh177 Looks great! Do we need the `div` here for some reason?

Reply:
Yes, the ```div``` element acts as a container, enclosing the ```span``` elements and preventing them from appearing side-by-side, which would happen if the ```div``` element was not used.
![hover-bug](https://user-images.githubusercontent.com/96344003/224799442-3b64e3f4-2807-471d-9dc2-1461f2a3438f.png)